analysis/model/make_model_input.R

```diff
@@ -94,450 +108,485 @@ for (i in 1:nrow(active_analyses)) {
     dplyr::rowwise() %>% 
     dplyr::mutate(end_date_outcome = min(end_date_outcome, out_date, na.rm = TRUE))
   
-  # Make model input: main -------------------------------------------------------
+  # Exclude individuals at index date
   
-  if (active_analyses$analysis[i]=="main") {
+  outcome <- active_analyses$outcome[i]
+
+  if (outcome == "out_date_cognitive_impairment_symptoms") {
+
+    input <- input %>%
+      filter(cov_bin_history_cog_imp_sympt == FALSE)
```

Comment:
The variables used for filtering should be removed at this stage as they now contain no information and are just making the dataset bigger.

Reply:
Removing the variables, remove covariates from the models (listed in the protocol). Dropping columns or not, produce files of the same size.

Reply:
This is because the dummy data is small but for 17mil rows, an extra variable will make more difference.